### PR TITLE
Remove the references to planet@python.org in the templates

### PR DESCRIPTION
**Pull request by matrixise**
See #287

### CODE REVIEW

**Comment by rochacbruno** [comment]
there is the html templates https://github.com/python/planet/blob/master/config/index.html.tmpl#L142

**Comment by matrixise** [comment]
@rochacbruno Updated, normally, it's okay, just need your confirmation and I will merge it asap

**Comment by matrixise** [comment]
@rochacbruno I will wait for your review, once approved, I will close this PR and the issue.

**Review by rochacbruno — APPROVED** [review]

**Pull request merged**
